lib/gis_robot_suite.rb

```diff
@@ -134,6 +134,35 @@ def self.locate_esri_metadata(dir, _opts = {})
     filename
   end
 
+  def self.locate_data_files(dir)
```

Comment:
Moved locating files into help methods so they could be used across multiple robots.